client/trino-cli/src/main/java/io/trino/cli/StatusPrinter.java

```diff
@@ -324,10 +324,16 @@ private void printQueryInfo(QueryStatusInfo results, WarningsPrinter warningsPri
             int progressWidth = (min(terminalWidth, 100) - 75) + 17; // progress bar is 17-42 characters wide
 
             if (stats.isScheduled()) {
-                String progressBar = formatProgressBar(progressWidth,
-                        stats.getCompletedSplits(),
-                        max(0, stats.getRunningSplits()),
-                        stats.getTotalSplits());
+                String progressBar;
+                if (stats.isFaultTolerantExecution()) {
+                    progressBar = formatProgressBar(progressWidth, progressPercentage, stats.getFaultTolerantExecutionRunningPercentage(), 100);
```

Comment:
nit: maybe  have `stats#getRunningPercentage` that has a condition based  on what execution mode is used?

Reply:
Not really feasible if we don't want to change the signature of `formatProgressBar`:
```
if (stats.isFaultTolerantExecution()) {
                    progressBar = formatProgressBar(progressWidth, progressPercentage, stats.getFaultTolerantExecutionRunningPercentage(), 100);
                }
                else {
                    progressBar = formatProgressBar(progressWidth,
                            stats.getCompletedSplits(),
                            max(0, stats.getRunningSplits()),
                            stats.getTotalSplits());
                }
```